src/util.cpp

```diff
@@ -719,8 +719,8 @@ fs::path GetDefaultDataDir()
         pathRet = fs::path("/");
     else
         pathRet = fs::path(pszHome);
-#ifdef MAC_OSX
-    // Mac
+#ifdef __APPLE__
```

Comment:
Do we for sure mean "Apple" in all of these changes and not just "macOS"? This would affect iOS as well, which seems likely incorrect here. I suspect that we want to keep using MAC_OSX (which we define ourselves) so that we can differentiate later if needed.

Reply:
Agree with @theuni

Reply:
I've dropped this change and replaced it with a scripted-diff that converts __APPLE__ usage to MAC_OSX.